samples/powerplay/build.gradle

```diff
@@ -0,0 +1,85 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+    id 'org.jetbrains.kotlin.plugin.compose'
+}
+
+android {
+    defaultConfig {
+        applicationId "com.example.powerplay"
```

Comment:
can you do find and replace for all instances of com.example.powerplay and change this to com.google.oboe.samples.powerplay? Otherwise, Android studio won't be able to find this if you open the samples directory

Reply:
When you do this, you need to also change the directory of your kotlin files and change com_example_powerplay_ to com_google_oboe_samples_powerplay_ for your jni code

Reply:
Ah ok, updated